src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -37,6 +37,8 @@ public ModelManager(ReadOnlyTaskManager taskManager, UserPrefs userPrefs) {
 
         versionedTaskManager = new VersionedTaskManager(taskManager);
         filteredTasks = new FilteredList<>(versionedTaskManager.getTaskList());
+        //Flag any task that are overdue as overdue
```

Comment:
typo

Reply:
@jiaxiu11 Resolved. Is the new comment sufficient?